TestApp/app/src/main/java/com/example/testapp/repository/MovieRepository.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.testapp.repository
+
```

Comment:
MovieRepo, MovieRepository, and IMovieRepo.. can u please differentiate between these three. One Repo class should be an interface and a class should be implementing this. that's it. Please recheck this structure. Please refer to the link provided: https://developer.android.com/jetpack/docs/guide